Plugins.properties

```diff
@@ -10,6 +10,7 @@ plugin=org.omegat.filters3.xml.xliff.XLIFFFilter \
     org.omegat.util.gui.HtmlMediaSupport \
     org.omegat.core.dictionaries.LingvoDSL \
     org.omegat.core.dictionaries.StarDict
+#   org.omegat.gui.main.NewUI
```

Comment:
Like this you have it inactive, coorect?
What is your plan for the future? Do you plan to move it to an external plugin?

Reply:
Now activate for review